python/setup.py

```diff
@@ -303,10 +303,10 @@ def get_packages(self):
         "dataclasses; python_version < '3.7'",
         "filelock",
         # Tracking issue: https://github.com/ray-project/ray/issues/30984
-        "grpcio >= 1.32.0, <= 1.49.1; python_version < '3.10' and sys_platform == 'darwin'",  # noqa
-        "grpcio >= 1.42.0, <= 1.49.1; python_version >= '3.10' and sys_platform == 'darwin'",  # noqa
-        "grpcio >= 1.32.0; python_version < '3.10' and sys_platform != 'darwin'",
-        "grpcio >= 1.42.0; python_version >= '3.10' and sys_platform != 'darwin'",
+        "grpcio >= 1.32.0, <= 1.49.1; python_version < '3.10'",  # noqa
+        "grpcio >= 1.42.0, <= 1.49.1; python_version >= '3.10'",  # noqa
```

Comment:
why not `<1.50.0` to automatically allow other patches to 1.49? per https://github.com/ray-project/ray/issues/30984

Reply:
oh, just saw that it was this way before. consider it a nit